setup-slave.sh

```diff
@@ -45,6 +45,21 @@ if [[ $instance_type == r3* ]]; then
   fi
 fi
 
+# Work around for hi1.4xlarge instances without pre-formatted ext3 disks
+if [[ $instance_type == hi1.4xlarge ]]; then
```

Comment:
Could we put this inside the same `if` clause as `r3` and `i2` (there was a recent commit for i2) ?